cumulus/parachains/integration-tests/emulated/assets/asset-hub-westend/src/tests/reserve_transfer.rs

```diff
@@ -36,10 +36,8 @@ fn relay_origin_assertions(t: RelayToSystemParaTest) {
 }
 
 fn system_para_dest_assertions_incomplete(_t: RelayToSystemParaTest) {
-	AssetHubWestend::assert_dmp_queue_incomplete(
-		Some(Weight::from_parts(1_000_000_000, 0)),
-		Some(Error::UntrustedReserveLocation),
-	);
+	// Errors with `UntrustedReserveLocation`, but the MQ pallet does not report back errors.
```

Comment:
Is there any way to get this error? Will this make things harder to debug?

Reply:
That is kind of a concern - yes. Problem is that if we want to get the exact error variant, then we need to use some aggregate error akin to the `RuntimeError` that spans across *all possible* implementations.  
Gav originally designed the MessageQueue interfaces to only return a `bool`, so the error needs to be emitted as event by the underlying logic. Not idea, but dont know how to make it better.

Reply:
True, it would be troublesome to support all exact variants. Maybe just returning a string wrapped in an error type?

Reply:
Maybe ask about it in the XCM channel. I think debuggability with the MessageQueue pallet is difficult in general.  
String could help, but is a bit unclean.